Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -9772,7 +9772,7 @@
         <ScoreMake value="75"/>
         <ScoreKill value="150"/>
         <ScoreResult value="BuildOrder"/>
-        <SubgroupPriority value="18"/>
+        <SubgroupPriority value="17"/>

```

Comment:
Impaler - burrow only - but this and Lurker should be above Marines/Wraiths (but under Banshees) due to how vital Burrow is for them